contents/docs/session-replay/network-recording.mdx

```diff
@@ -35,6 +35,8 @@ When request capture is enabled you can also enabled payload and body capture, t
 
 ## Sensitive information
 
+We automatically scrub some sensitive information from network headers, but your request or response payloads could contain sensitive data, you should provide a function to mask the captured data when you initialise PostHog.
```

Comment:
```suggestion
We automatically scrub some sensitive information from network headers, but your request or response payloads could contain sensitive data, you should provide a function to mask the captured data when you Initialize PostHog.
```

IIRC we prefer the US form.

Reply:
damn, I don't even see it... too good at spelling is finally a curse 🤣

Reply:
```suggestion
We automatically scrub some sensitive information from network headers, but your request or response payloads could contain sensitive data, you should provide a function to mask the captured data when you initialize PostHog.
```